server.go

```diff
@@ -67,6 +90,83 @@ func NoDirListing(h http.Handler) http.HandlerFunc {
 	})
 }
 
+func initBlogs() {
+	blogs = make([]BlogItem, 0)
+  blogs = append(blogs, 
+		createBlogEntry("A green landscape", "/img/landscape5.jpg"),	
+		createBlogEntry("A mountain landscape", "/img/landscape6.jpg"),	
+		createBlogEntry("A lake landscape", "/img/landscape1.jpg"),	
+		createBlogEntry("A trees landscape", "/img/landscape2.jpg"),
+		createBlogEntry("A village landscape", "/img/landscape3.jpg"),	
+		createBlogEntry("A canyon", "/img/landscape4.jpg"),	
+		createBlogEntry("A dog", "/img/Shetland_Sheepdog.jpg"),	
+		createBlogEntry("A phone", "/img/silver_nexus.png"),
+		createBlogEntry("Some Sushi", "/img/image3.jpg"),
+		createBlogEntry("Hi Gopher!", "/img/gopher.png"))
+}
+
+func createBlogEntry(text string, imagePath string) BlogItem {
+	return BlogItem{Text: text, Image: imagePath, Timestamp: currentTimestamp()}
+}
+
+//create a clone of the blog item, instead of pointing
+func cloneBlogItem(blogItem BlogItem) BlogItem{
+	return createBlogEntry(blogItem.Text, blogItem.Image)
+}
+
+//returns a map of blogs which contains a number of items specified by the parameter
+func returnMapOfBlogOfSize(size int) []BlogItem {
+		ArrayOfBlogsToReturn := make([]BlogItem, 0)
+		if size > len(blogs) {
+			size = len(blogs)
+		}
+		for i := 0; i < size; i++ {
+    	ArrayOfBlogsToReturn = append(ArrayOfBlogsToReturn, cloneBlogItem(blogs[i]))
+		}
+		return ArrayOfBlogsToReturn
+}
+
+func updateScore(score string) Score{
```

Comment:
Do you mean adding more scores? E.g.: at the moment there is only one match, going on, would you like to have more than one when the page loads for the first time and then updating the values at every iteration?